src/components/expanded-state/chart/ChartExpandedStateHeader.js

```diff
@@ -147,6 +147,7 @@ export default function ChartExpandedStateHeader({
             isScrubbing={isScrubbing}
             priceRef={priceRef}
             priceValue={defaultPriceValue}
+            ratio={ratio}
```

Comment:
I used this prop in the component as a base since we calculate it on the JS thread. but somehow i didn't commit this line in the first place